### ISSUE TITLE
feature: Drop down Title Bar [Request]

### ISSUE THREAD

**Issue by Genxster1998** [boringdroid/boringdroid#1]
Thanks for your contribution to android on pc,
You have done what Phoenix Os kept closed (modified code of xposed halo floating window) while Google in chrome os has handled it to it's ash windows management system leaving freeform window a mere joke which is best to left alone by other Dev's like Samsung Dex , Phoenix os , Openth os.

What about drop down title bar on arrow reaching upper end of screen  for a maximised window ? It's difficult but it can be implemented right ? I tried bliss x86 , there's no way back to windowed mode after fullscreen or to close or minimize.
Also what about replace min,max,close button icons (confusing) and reducing width of title bar (decorview).

Thanks , keep it up ! 
Have a great day

**Comment by utzcoz** [comment]
@Genxster1998 thanks your suggestion. It can be one thing we can do in later release. There is a keyboard shortcut - `F11` - to switch window from fullscreen to freeform window. At least, bliss x86 11 recent release version has integrated it.

**Comment by utzcoz** [comment]
I am starting to work on it.

**Comment by utzcoz** [comment]
I added the peek caption for legacy caption. For modern WMShell and desktop mode, it uses real maximized not fullscreen.
Example: ![peek caption](https://github.com/boringdroid/boringdroid/blob/boringdroid-14.0.0/images/peek-caption.png)

Maybe it will be deleted when I think boringdroid only focus on modern WMShell caption.

At least, the basic version has been implemented but it requires a lot of properties to enable it.

**Comment by Genxster1998** [comment]
What's modern shell caption , does it retain the titlebar in maximised window state? Btw , it's been a really long time, good to see , you still tend to your passion projects.

**Comment by utzcoz** [comment]
Yeah. Its maximize state is just using the bounds of display and keeping caption bar in my testing on boringdroid 14.

**Comment by utzcoz** [comment]
What I mean "modern caption" is the caption implementation based on WMShell not legacy caption in DecorView.
